frontend/src/components/TransactionRows/CollapseIcon.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <div class="collapse-icon">
+    <b-icon
+      :icon="visible ? 'caret-up-square' : 'caret-down-square'"
+      :class="visible ? 'text-black' : 'text-muted'"
+    />
+  </div>
+</template>
+<script>
+export default {
+  name: 'Collapse-Icon',
+  props: {
+    visible: {
+      type: Boolean,
+    },
```

Comment:
is `sichtbar` the right wording here? Isn't it rather `enabled`?

Reply:
Also: required? default?

Reply:
geändert